backend/api/index.ts

```diff
@@ -17,20 +17,23 @@ app.use(express.json());
 app.use(cookieParser());
 
 // declare a route with a response
-app.get("/", (req, res) => {
+app.get("/api", (req, res) => {
   res.send("Server running");
 });
 
-app.post("/signup", emailCheck, passwordCheck, signup);
+app.post("/api/signup", emailCheck, passwordCheck, signup);
 
-app.post("/login", loginController);
+app.post("/api/login", loginController);
 
-app.post("/logout", logoutController);
+app.post("/api/logout", logoutController);
 
 connect().then(() => {
-  // start the server
-  const PORT = process.env.BACK_PORT || 8081;
-  app.listen(PORT, () => {
-    console.log(`server running : http://localhost:${PORT}`);
-  });
+  console.log("connected to database");
```

Comment:
Hi Valeria, if you would like to run the server locally, you can comment out the lines of code I have commented out and comment out "module.exports = app;" but if you would like to have the remote server on vercel we have to do this. This is what vercel requires as it is serverless.

Is it best practice to have api folder instead of src?

Yes it is best practice according to the documentation on vercel for a typescript app.